### DISCUSSION TITLE
Issues with Kitsu

### DISCUSSION THREAD

**Discussion by Goldenfreddy0703** [Goldenfreddy0703/plugin.video.kaito.beta#43] [General]
Hey @Gujal00 , i didn't know where to message you so i thought i should message you here, do you think you can test out Kitsu Watchlist Update Sync. I don't think that's working and i could use your help on that.

**Comment by Gujal00** [comment]
Do you have a error log so that I can have a starting point of the failure

**Reply by Goldenfreddy0703** [reply]
Hey so I don't but when I get home, I will provide one. Should be home in about 30min.

**Comment by Goldenfreddy0703** [comment]
![image](https://user-images.githubusercontent.com/62456796/184569504-c0ab4846-b844-4859-abfd-924f54edc2cd.png)
Hey sorry i took forever but here is the error for Kitsu, The problem is just updating the watchlist so far.

**Reply by Gujal00** [reply]
looks like it will be hard to debug without an active kitsu account which I do not have.
Can you try replace kitsu.py file in the WatchlistFlavor folder with the attached one and try. If it still fails, I will get an account and debug further
[Kitsu.zip](https://github.com/Goldenfreddy0703/plugin.video.kaito.beta/files/9335713/Kitsu.zip)

**Comment by Goldenfreddy0703** [comment]
Hey, sorry i was asleep but looks like that kitsu.py worked, all watchlist seem to be working now. Thank you very much for fixing kitsu and for fixing all the watchlist stuff.
